docs/source/orderer/ordering_service.md

```diff
@@ -185,7 +185,17 @@ between ordering service nodes.
 For information about how to stand up an ordering node (regardless of the
 implementation the node will be used in), check out [our documentation on deploying a production ordering service](../deployorderer/ordererplan.html).
 
-* **Raft** (recommended)
+* **BFT** 
+  New as of v3.0, the Byzantine Fault Tolerant (BFT) ordering service, as its name implies,
+  can withstand not only crash failures, but also a subset of nodes behaving maliciously.
+  An example of malicious behavior is a leader node sending different blocks with the same sequence number,
+  hence creating a fork in the blockchain. An ordering service that is only Crash Fault Tolerant (CFT) 
+  can not protect against such a behavior. 
+  A Byzantine Fault Tolerant ordering service can withstand up and not including to a third of its nodes misbehaving
+  (either crashing or deviating from the protocol).
+  More information about the BFT protocol can be found in its corresponding [paper](https://arxiv.org/abs/2107.06922). 
+
```

Comment:
This is a good intro. For Raft, this document also has a decent amount of conceptual information about the protocol. Are we intending to add similar conceptual information about SmartBFT?

Reply:
can  you be more specific?

Reply:
Take a look at https://hyperledger-fabric.readthedocs.io/en/latest/orderer/ordering_service.html#raft. The bottom half of that document provides a conceptual description of how Raft works in the Fabric ordering service. This is missing for SmartBFT.